### PR TITLE
Automates creation of OAuth token with 2FA using GitHub

### PR DESCRIPTION
**Pull request by codenamev**
Automates creation of OAuth token with 2FA using GitHub

Pending merge of https://github.com/peter-murach/github/pull/213

### CODE REVIEW

File: lib/git_reflow/git_server/git_hub.rb [hunk 1]
```diff
@@ -61,33 +63,11 @@ def authenticate(options = {silent: false})
...
-              self.class.user = gh_user
-
-              if previous_authorizations.any?
-                authorization = previous_authorizations.last
-              else
-                authorization = github_authorizations.create scopes: ['repo'], note: "git-reflow (#{run('hostname', loud: false).strip})"
-              end
-
-              self.class.oauth_token = authorization.token
-              puts "\nYour GitHub account was successfully setup!"
```
Comment (nhance) [inline lib/git_reflow/git_server/git_hub.rb]: Do we want to remove this?

File: lib/git_reflow/git_server/git_hub.rb [hunk 2]
```diff
@@ -61,33 +63,11 @@ def authenticate(options = {silent: false})
             end
 
             self.class.oauth_token = authorization.token
-            puts "\nYour GitHub account was successfully setup!"
```
Comment (nhance) [inline lib/git_reflow/git_server/git_hub.rb]: How come this goes away?

Comment (codenamev) [inline lib/git_reflow/git_server/git_hub.rb]: there is an `else` to the `begin` that catches both cases we want

**Comment by nhance** [comment]
lgtm

**Pull request closed without merging**